src/electron/views/device-connect-view/components/window-title.scss

```
@@ -4,7 +4,23 @@
 @import '../../../../common/styles/fonts.scss';
 
 .window-title {
-    padding: 6px 9px;
+    width: 100%;
+    height: 32px;
+    display: flex;
+    align-items: center;
+
```

```suggestion
    -webkit-app-region: drag;
```

This should already have been in `title-bar` but was not. Suggesting adding it here, but doesn't need to be in this PR.